requirements_dev.txt

```diff
@@ -11,6 +11,6 @@ pip==18.1
 psycopg2-binary==2.7.5
 pyflakes==2.0.0
 pycodestyle==2.4.0
-setuptools==40.5.0
+setuptools==40.6.1
```

Comment:
It's not normal to include setuptools in the requirements, is it?

I'd expect it to be installed alongside `pip`.

Reply:
Yes, it is normally installed but most of the time it's outdated. On my reference Ubuntu 14 system, setuptools is version 3.4.3 and pip is version 1.5.4.

Plus, this is the `requirements_dev.txt` which is only needed if you are doing development on the library. It's not used when it is installed as a package.